docker/registry/entry.sh

```diff
@@ -1,7 +1,11 @@
 #!/bin/bash
 
 if [ ! -e /registry/config.yml ]; then
-  sed -e "s|DOCKER_HOST|${DOCKER_HOST}|g" /registry/config.yml.template > /registry/config.yml
+  sed -e "{
+    s|DOCKER_HOST|${DOCKER_HOST}|g;
+    s|PORTUS_DB_HOST|${PORTUS_DB_HOST}|g;
```

Comment:
The config.yml file is not using the `PORTUS_DB_HOST` variable. You can remove this line.